packages/cli/src/databases/migrations/postgresdb/1677501636754-CreateVariables.ts

```diff
@@ -9,7 +9,7 @@ export class CreateVariables1677501636754 implements MigrationInterface {
 		const tablePrefix = getTablePrefix();
 
 		await queryRunner.query(`
-			CREATE TABLE public.variables (
+			CREATE TABLE ${tablePrefix}variables (
```

Comment:
Should we wrap in double quotes for case sensitivity?

Reply:
we ideally should. but if someone is using a case sensitive table-prefix, we have previous migrations that would fail. handling of case-sensitive table-prefix needs to be a separate PR.